src/reanimated2/commonTypes.ts

```diff
@@ -94,30 +94,92 @@ export type MapperRegistry = {
   stop: (mapperID: number) => void;
 };
 
-export type Context = Record<string, unknown>;
+export type WorkletClosure = Record<string, unknown>;
 
-export interface WorkletFunction {
-  _closure?: Context;
+interface ReleaseInitData {
```

Comment:
1. I think we should name this type `WorkletInitData`, it sounds more complete then, not to mention it's more analogous to `WorkletClosure`.
2. I think we should put `Dev`/`Release` at the end of the name. People usually start reading from the left and `WorkletInitDataDev` looks more similar to `WorkletInitDataRelease` than `DevWorkletInitData` and `ReleaseWorkletInitData`, not to mention autocomplete.